### ISSUE TITLE
Error message for failing test not displayed in report

### ISSUE THREAD

**Issue by rjmartell** [opengeospatial/teamengine#136]
__Affected version: 4.4-rc1__

If an exception (RuntimeException) occurred while executing a test method in a TestNG suite, the error message does not appear in the HTML report. The "Reason For Test Failure" entry is empty even though the TestNG results file includes non-empty message content.

__To reproduce:__

Stop Tomcat and rename the EPSG database directory located at `-Dderby.system.home` (a new one will be automatically created, but without the EPSG data). Restart and run the GML test suite; validate the GML document referenced below.

[ProtectedSites-1.txt](https://github.com/opengeospatial/teamengine/files/23664/ProtectedSites-1.txt)

TestNG results:
```
<test-method status="FAIL" name="validSurfaceOrientation">
  <exception class="java.lang.RuntimeException">
    <message>
      <![CDATA[Failed to create coordinate transformer.]]>
    </message>
  </exception>
</test-method>
```

HTML report:

![no-error-message](https://cloud.githubusercontent.com/assets/1653506/10828889/36ed09a2-7e34-11e5-8673-8dcf977364e2.png)

Perhaps the report generator ignores error messages that are not due to failing assertions?

**Comment by rjmartell** [comment]
This is a regression. In TE 4.3 the exception message was displayed in the HTML report: 
"java.lang.RuntimeException: Failed to create coordinate transformer."

**Comment by rjmartell** [comment]
Observed in both Windows (7 Pro SP1) and Linux (RHEL 7.1) environments with Tomcat 7.0 and Oracle JDK 8.

**Comment by keshav-nangare** [comment]
The issue "Error message for failing test not displayed in report #136 " (4.4-rc1 fix) is resolved with commit: 1ab9f7f4402d168e023ffa0f393213ee34869bcf
